lizard_auth_server/templates/lizard_auth_server/activation_email_en.txt

```diff
@@ -1,26 +1,15 @@
 Dear {{ name }},
 
-
-Thank you for requesting an account for one of our products.
+This is an invitation for 3Di. 
 You can activate your account and enter you chosen password via this activation link:
 {{ activation_url }} *
 
-
 Your username is: {{ username }}.
-After activation, you can log in with your username and password at {{ portal_url }}.
-
-To have a quick start, you can find the documentation and release notes of
-Lizard and 3Di here:
-
-- https://docs.lizard.net/
-- https://docs.3di.lizard.net/
-
-If you have any questions, do not hesitate to contact us. You can do that by
-sending an email to servicedesk@nelen-schuurmans.nl
+After activation, you can log in with your username and password at https://3di.live/
```

Comment:
De test faalt omdat er nu alleen hardcoded '3di.live' staat i.p.v. de url van de portal.
In de test wordt reinout.vanrees.org verwacht :-)
Ik ga even kijken of ik dat kan fixen, dat is dan gelijk een test of het met visual studio code werkt.